modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -401,6 +401,19 @@ public void update(
 		super.update(objectEntries, parameters);
 	}
 
+	@Override
+	protected Long getPermissionCheckerGroupId(Object id) throws Exception {
+		return contextCompany.getCompanyId();
```

Comment:
Is the `companyId` what we have to return? I have seen in the PermissionUtil class that the id returned by this method is being used to get the group